### PR TITLE
Update summer-school-2021.yaml

### PR DESCRIPTION
**Pull request by justjosie**
Updating "Learn More" URL to have correct destination link

### CODE REVIEW

File: new-content/events/summer-school-2021.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ header:
     - 'Speakers will be announced soon!'
   cta:
     label: 'Learn more'
-    url: 'https://qiskit.org'
+    url: 'http://qisk.it/qgss2021blog_website_source'
```
Comment (techtolentino) [inline new-content/events/summer-school-2021.yaml:15]: Hey @justjosie 

Can we please update this to `https`

Thank you

File: new-content/events/summer-school-2021.yaml [hunk 2]
```diff
@@ -21,7 +21,7 @@ header:
     location: 'Online'
     date: 'July 12 - 23, 2021'
     time:
-    to: 'https://qiskit.org'
+    to: 'http://qisk.it/qgss2021blog_website_source'
```
Comment (techtolentino) [inline new-content/events/summer-school-2021.yaml:24]: same please

Comment (techtolentino) [reply new-content/events/summer-school-2021.yaml:24]: Okay, I see that it redirects to a secure link... and I also see that our link to metal also uses `http://qisk.it/metal`

I'm good with it.

**Review by techtolentino — APPROVED** [review]

**Comment by korgan00** [comment]
Changes moved to #1717

**Pull request closed without merging**